gradle/plugins/src/main/kotlin/com.hedera.gradle.nexus-publish.gradle.kts

```diff
@@ -44,7 +44,8 @@ tasks.named("closeSonatypeStagingRepository") {
 
 tasks.named("releaseMavenCentral") {
     group = "release"
-    dependsOn(tasks.closeAndReleaseStagingRepository)
+//    dependsOn(tasks.closeAndReleaseStagingRepository)
+    dependsOn(tasks.named("closeSonatypeStagingRepository"))
```

Comment:
After merging #15282 this can be controlled via a parameter to the `./gradlew` call in the GH actions flow.

- `-PpublishTestRelease=true` will use `closeSonatypeStagingRepository `
- `-PpublishTestRelease=false` (or no parameter) will use `closeAndReleaseStagingRepository`

This and the other parameters will be documented here: https://github.com/hiero-ledger/hiero-gradle-conventions/tree/documentation?tab=readme-ov-file#in-github-actions (docs are wip right now)